deepspeed/runtime/activation_checkpointing/checkpointing.py

```diff
@@ -348,6 +353,132 @@ def merge_tensors(tensor_objects, non_tensor_objects, tensor_flags):
     return tuple(merged_objects)
 
 
+def is_activation_to_checkpoint(item):
+    """
+        Is an activation to be checkpointed
+    """
+    global mp_size
+    return torch.is_tensor(item) and item.is_floating_point() and item.numel() >= mp_size

```

Comment:
don't know why using condition of "numel() > mp_size", could you add some comments here?